extensions/investspec/internationalization/locales/es/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,125 @@
+# Spanish translations for InVEST.
+# Copyright (C) 2021 Natural Capital Project
+# This file is distributed under the same license as the InVEST project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: InVEST 3.10\n"
+"Report-Msgid-Bugs-To: esoth@stanford.edu\n"
+"POT-Creation-Date: 2021-11-30 13:20-0800\n"
+"PO-Revision-Date: 2021-11-30 13:21-0800\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: es\n"
```

Comment:
I don't think I saw this key in the other PO files, should we update the other PO files for consistency?

Reply:
I think it's because this PO file is created by pybabel while the others are created by sphinx-intl. I guess they have slightly different headers (all the header fields are optional according to the GNU gettext standard). I don't feel it really matters since the language is defined in the path but can add it to the other PO files if you prefer

Reply:
I don't think I'm familiar enough to have a preference! But I think you're right, language is embedded in the path so not necessary.